examples/minimum/main.tf

```diff
@@ -0,0 +1,17 @@
+module "minimum" {
+  source                 = "./../../"
+  name                   = "example-minimum"
+  description            = "Terraform sample github repository "
+  visibility             = "private"
+  require_signed_commits = true

```

Comment:
For the min make it the smallest possible, for example, you don't have to specify any options beyond name and desciption.